### PR TITLE
Mse ctl 0.8

### PR DESCRIPTION
**Pull request by heavenboy8**
Adding an example about how to use paths in the enclave

Also be compliant with new config file from mse-cli 0.8

### CODE REVIEW

File: path/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Path
+
+Basic example of how to use dedicated paths in an mse application.
+
+ This example works with a self-signed certificate on a fully zero trust environment and containing:
```
Comment (ThibsG) [inline path/README.md]:
```suggestion
 This example works with a self-signed certificate on a fully zero trust environment. This example contains:
```

File: path/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- The mse app config file
+- A secret file
+- Python tests
+
+## Test it locally
+
+On a first terminal, run:
+
+```console
+$ # From path directory
```
Comment (ThibsG) [inline path/README.md]:
```suggestion
$ # From `path` example directory
```

File: path/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+$ cd mse_src
+$ SECRETS_PATH=../secrets.json flask run
+```
+
+You can also run instead: 
+
+```console
+$ # From path directory
```
Comment (ThibsG) [inline path/README.md]:
```suggestion
$ # From `path` example directory
```

File: path/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+$ mse test
+```
+
+On a second terminal, run:
+
+```console
+$ # From path directory
```
Comment (ThibsG) [inline path/README.md]:
```suggestion
$ # From `path` example directory
```

File: path/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+```console
+$ mse verify --skip-fingerprint "<uuid.cosmian.app>"
+```
+
+You can now query the microservice:
+
+```sh
+$ curl https://<uuid.cosmian.app>/ --cacert cert.pem
+```
```
Comment (ThibsG) [inline path/README.md]: we have `app_domain_name` or `<uuid.cosmian.app>`, but shouldn't we use the same domain name in these 3 command lines?

File: path/mse_src/app.py [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+@app.route('/whoami')
+def whoami():
+    """A simple example manipulating secrets."""
+    secrets = json.loads(Path(os.getenv("SECRETS_PATH")).read_text())
+    return secrets["login"]
+
+
+@app.post('/')
+def write_date():
+    """A simple example of file writting."""
```
Comment (ThibsG) [inline path/mse_src/app.py]:
```suggestion
    """A simple example of file writing."""
```

**Review by ThibsG — APPROVED** [review]

File: path/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+```console
+$ mse deploy 
+```
+
+Your application is now ready to be used
+
+## Test it
+
+```console
+$ TEST_REMOTE_URL="https://<app_domain_name>" pytest
```
Comment (heavenboy8) [inline path/README.md]:
```suggestion
$ TEST_REMOTE_URL="https://<uuid.cosmian.app>" pytest
```

File: path/README.md (hunk 5 above)
Comment (heavenboy8) [reply path/README.md]: true and fixed

**Pull request merged**